web/css/source/variables/custom/_colors.less

```diff
@@ -0,0 +1,4 @@
+// Colors
+@color-orange-mixed-with-white: #ffb89e;
+@new-awesome-orange-color: #e8a535;
```

Comment:
magento's practice to start the color variables from the `@color-...`. follow that as well